v_4_6_0/master_template.go

```diff
@@ -323,6 +323,51 @@ systemd:
       --v=2"
       ExecStop=-/usr/bin/docker stop -t 10 $NAME
       ExecStopPost=-/usr/bin/docker rm -f $NAME
+      [Install]
+      WantedBy=multi-user.target
+  - name: node-exporter.service
+    enabled: true
+    contents: |
+      [Unit]
+      Description=Node Exporter
```

Comment:
On worker we require `docker.service` and other units before to start. here no. any reasons for that?

Reply:
typo